src/treemodel/schema.js

```diff
@@ -274,7 +331,7 @@ export default class Schema {
 	 *		// Check whether bold text is allowed in header element.
 	 *		let query = {
 	 *			name: '$text',
-	 *			attribute: 'bold',
+	 *			attributes: 'bold',
 	 *			inside: 'header'
 	 *		};
 	 *		if ( schema.checkQuery( query ) ) { ... }
```

Comment:
I feel that the API is now overcomplicated. There is `checkAtPosition`, `checkInElements ` and `checkQuery` methods. My idea is to rename `checkQuery` to simple `check` and make it a little bit smarter so it check if `this.hasItem( name )`  and what is the type of `inside`, accepting array of elements, array of strings and position. Thanks to such change user would not need to think which method he should use.